admin-frontend/src/providers/authProvider.js

```diff
@@ -23,6 +23,8 @@ export const authProvider = (type, params) => {
             })
             .then(() => httpClient(api.youthWorker.self, { method: 'GET' }))
             .then((response) => {
+                localStorage.setItem('lastLoggedInAdminName', response.firstName);
+                localStorage.setItem('lastLoggedInAdminMainYouthClub', response.mainYouthClub || '');
```

Comment:
Pretty hacky 😬

`admin-frontend` seems to have redux for some things, so maybe it would be possible to use that instead of localStorage?